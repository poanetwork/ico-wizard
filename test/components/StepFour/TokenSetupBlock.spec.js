import React from 'react'
import Adapter from 'enzyme-adapter-react-15'
import { configure, shallow } from 'enzyme'
import { TokenSetupBlock } from '../../../src/components/StepFour/TokenSetupBlock'

configure({ adapter: new Adapter() })

describe('TokenSetupBlock', () => {
  it(`should render the component `, () => {
    const data = {
      tokenStore: {
        name: 'This is a valid name',
        ticker: 'TTK',
        decimals: '14',
        supply: '0'
      }
    }
    const wrapper = shallow(<TokenSetupBlock {...data} />)

    expect(wrapper).toMatchSnapshot()
  })
})
