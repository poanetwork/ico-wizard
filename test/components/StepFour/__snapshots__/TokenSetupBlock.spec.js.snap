// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TokenSetupBlock should render the component  1`] = `
<div
  className="ts-TokenSetupBlock"
>
  <DisplayField
    description="The name of your token. Will be used by Etherscan and other token browsers."
    title="Name"
    value="This is a valid name"
  />
  <DisplayField
    description="The five letter ticker for your token."
    extraClassName="pb-DisplayField_Value-uppercase"
    title="Ticker"
    value="TTK"
  />
  <DisplayField
    description="How your token is divisible."
    title="Decimals"
    value="14"
  />
  <DisplayField
    description="Total token initial supply."
    title="Supply"
    value="0"
  />
</div>
`;
